translation-web/src/main/java/eu/europeana/api/translation/web/TranslationController.java

```diff
@@ -10,24 +13,28 @@
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
+import eu.europeana.api.commons.error.EuropeanaI18nApiException;
 import eu.europeana.api.commons.web.http.HttpHeaders;
 import eu.europeana.api.commons.web.model.vocabulary.Operations;
 import eu.europeana.api.translation.definitions.language.LanguagePair;
 import eu.europeana.api.translation.definitions.vocabulary.TranslationAppConstants;
 import eu.europeana.api.translation.model.TranslationRequest;
 import eu.europeana.api.translation.model.TranslationResponse;
 import eu.europeana.api.translation.web.exception.ParamValidationException;
+import eu.europeana.api.translation.web.service.RedisCacheService;
 import eu.europeana.api.translation.web.service.TranslationWebService;
 import io.swagger.v3.oas.annotations.tags.Tag;
 @RestController
 @Tag(name = "Translation endpoint", description = "Perform text translation")
 public class TranslationController extends BaseRest {
 
   private final TranslationWebService translationService;
+  private final RedisCacheService redisCacheService;
 
   @Autowired
-  public TranslationController(TranslationWebService translationService) {
+  public TranslationController(TranslationWebService translationService, RedisCacheService redisCacheService) {
```

Comment:
the caching service is not needed in the controller but in the TranslationWebService